diff -urN network-manager-applet-1.8.22.orig/meson.build network-manager-applet-1.8.22/meson.build
--- network-manager-applet-1.8.22.orig/meson.build	2019-06-28 22:36:03.509611322 +0200
+++ network-manager-applet-1.8.22/meson.build	2019-06-28 22:36:16.499610928 +0200
@@ -477,7 +477,7 @@
     configuration: ent_conf
   )
 
-  doc_path = join_paths(nma_datadir, 'gtk-doc', 'html', mod_name)
+  doc_path = join_paths(nma_datadir, 'doc', 'gtk-doc', 'html', mod_name)
 
   gnome.gtkdoc(
     mod_name,
